utils/send_json.py

```diff
@@ -0,0 +1,17 @@
+import requests
+from typing import Dict
+
+
+def send(data: Dict, job_id: str, file_id: str) -> object:
+    # Takes in a dictionary, appends job and file id and  converts into json then sends to url.
+    # Returns return code from the site in form of request object.
+
+    url = "https://api-smart-invoice-dev.b-mind.pl/api/v1/webhook/ml/send"
```

Comment:
Let's not hard-code any settings in the code. Instead, we'll use one of the available configuration libraries. Please check which one is easier to use:
* [simple-settings](https://pypi.org/project/simple-settings/)
* [pytohn-dotenv](https://github.com/theskumar/python-dotenv)